pkg/apis/kudo/v1alpha1/framework_types.go

```diff
@@ -21,10 +21,16 @@ import (
 
 // FrameworkSpec defines the desired state of Framework
 type FrameworkSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	Description       string       `json:"description"`
+	KudoVersion       string       `json:"kudoVersion"`
+	KubernetesVersion string       `json:"kubeVersion"`
+	Maintainers       []Maintainer `json:"maintainers"`
+	URL               string       `json:"url"`
```

Comment:
What about `name` and `version`?

Reply:
Those are captured in the `metadata.name` field and the `FrameworkVersion` objects